### PR TITLE
Add New Test case

### PR DESCRIPTION
**Pull request by YangCheng0121**

### CODE REVIEW

**Comment by YangCheng0121** [comment]
image

File: xray_wrapper_test.go [hunk 1]
```diff
@@ -1,123 +0,0 @@
-package libXray_test
```
Comment (yuhan6665) [inline xray_wrapper_test.go:1]: why remove it?

Comment (YangCheng0121) [reply xray_wrapper_test.go:1]: To add more test cases for convenience in the future

Comment (YangCheng0121) [reply xray_wrapper_test.go:1]: I plan to add test cases for all functions

Comment (yuhan6665) [reply xray_wrapper_test.go:1]: Maybe this can wait for more changes before merging

Comment (YangCheng0121) [reply xray_wrapper_test.go:1]: OK

**Pull request closed without merging**